src/re/correlated_field.py

```diff
@@ -27,6 +29,96 @@ def hartley(p, axes=None):
     return add_or_sub(tmp.real, tmp.imag)
 
 
+def get_sht(nside, axis, lmax, mmax, nthreads):
+    from jaxbind.contrib.jaxducc0 import get_healpix_sht
```

Comment:
Done. I added it to re directly. Could you double check that this is the intended place? I'm happy to leave it there, having a designated optional section for spherical harmonics might also be a viable option.

Reply:
I think a dedicated optional section might make more sense here since we also need to pull in ducc0, a dependency for which we also introduced a new optional dependency section.